grails-app/migrations/0.8.x/changelog-2021-12-14-1140-add-submit-request-activity.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+  <!-- selection for location_type_code = 'DEPOT' AND supported_activities_string = 'SUBMIT_REQUEST'-->
```

Comment:
Add a note, that for code depot the id of location type is `2` (if you are leaving the id conditional)